src/main/scala/is/hail/expr/ir/functions/MathFunctions.scala

```diff
@@ -0,0 +1,102 @@
+package is.hail.expr.ir.functions
+
+import is.hail.expr.ir._
+import is.hail.expr.types._
+import is.hail.stats
+import org.apache.commons.math3.special.Gamma
+
+object MathFunctions extends RegistryFunctions {
+  // FIXME can't figure out how to get the instance of a package object
```

Comment:
I think you can just call e.g. `Code.invokeScalaObject(Class("scala.math.package$"), ....)`?

Reply:
Yes, looks like.

I was trying to do it within Scala, but you just can't seem to get the object corresponding to a package object from within the language.

Reply:
OK, fixed.

Reply:
FYI, I changed registerScalaFunction to take the class instead of the object (since I still can't get a handle on the object for a package object).